Lesson34/selcan/lesson34-app/src/components/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import "./styles.css";
+
+// TODO: Update component so that it's possible to change user name and save new user name in the UserContext
+
+export const UserDetailsForm = ({ incomingUsername }) => {
```

Comment:
`incomingUsername` prop is not used in this component, remove. Make sure to remove it also from places where `UserDetailsForm` component is used (see Account component).